docs/dictionary/control_st/return.lcdoc

```diff
@@ -2,39 +2,142 @@ Name: return
 
 Type: control structure
 
-Syntax: return <value> 
+Syntax: return [ value | error ] <value>
```

Comment:
Change to `Syntax: return [{ value | error }] <value>` should pass the docs test